### ISSUE TITLE
Run `terraform validate` for e2e tests

### ISSUE THREAD

**Issue by MarkMcCulloh** [winglang/wing#4288]
### Feature Spec

`turbo test -F=hangar` includes verification of terraform output for `wing compile -t tf-aws ...` scenarios via `terraform validate`.

### Use Cases

Quickly catches a small subset of terraform issues that wing could generate.

### Implementation Notes

Should this be built-in to `wing compile -t tf-aws`?

### Component

Contributor Experience

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by staycoolcall911** [comment]
>Should this be built-in to wing compile -t tf-aws?

Yes, let's try to add it to `wing compile` and see the impact on performance and the overall experience
